libs/iamutil/user.go

```diff
@@ -18,3 +18,13 @@ func GetShortUserName(user *iam.User) string {
 	local, _, _ := strings.Cut(name, "@")
 	return textutil.NormalizeString(local)
 }
+
+// GetShortUserDnsName returns a dns-friendly for of the user name based on short name
```

Comment:
Any alternatives for `dns_name`? Before reading the code I thought it somehow related to user's own domain.

How about "safe_short_name" since we already have "short_name"? or "domain_safe_short_name". Long but explicit.

Reply:
changed the variable name in [a7e45e7](https://github.com/databricks/cli/pull/3623/commits/a7e45e78ac3f7af89929173aed929bfd13a5dd7e)